src/components/audio_instructions_web/web_audio_player.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * WebAudioPlayer
+ * Author: Simon Franzen
+ */
+
+/**
+ * Constructor
+ *
+ * Initialize audio player and bind to canplaythrough and
+ * timeupdate from audio player
+ */
+
+var WebAudioPlayer = function (options) {
+
+    this.options = options || {};
+
+    this.audioData = options.audioData;
+
+    //check if browser can play audio file
+    this.canPlayAudio = ('speechSynthesis' in window);
+
+    this.enabled = false;
+
+};
+
+WebAudioPlayer.prototype.options = null;
+
+WebAudioPlayer.prototype.canPlayAudio = null;
+
+/**
+ * Play a string with a street name
+ *
+ * .
+ */
+WebAudioPlayer.prototype.play = function (directionCode, street) {
+
+    if (!this.enabled) return false;
+
+    if (!this.canPlayAudio) return false;
+
+    direction =  this.audioData[directionCode];
+    if (typeof direction === 'undefined') {
+        console.log('no phrase for ' + directionCode);
+        return false;
+    }
+
+    if (typeof street !== undefined) {
+        direction = direction.replace('!', 'on to ' + street);
```

Comment:
I think we should use another marker that allows this class to do more interpolation in the future.
Curly brackets are very common, I think this would work here too: `in 100m turn hard right {{street}}`

Reply:
done - also remove "on to" hard coded in player.